lms/djangoapps/courseware/tests/test_date_summary.py

```diff
@@ -711,37 +709,22 @@ def test_render_date_string_past(self, delta, expected_date_string):
             block = VerificationDeadlineDate(course, user)
             assert block.relative_datestring == expected_date_string
 
-    @ddt.data(
-        ('info', True),
-        ('info', False),
-        ('openedx.course_experience.course_home', True),
-        ('openedx.course_experience.course_home', False),
-    )
-    @ddt.unpack
+    @ddt.data('info', 'openedx.course_experience.course_home')
```

Comment:
Why this change? I understand we don't need to override `COURSE_HOME_MICROFRONTEND_DATES_TAB` anymore, but why not keep the "overriding `COURSE_HOME_MICROFRONTEND`" part of this test?

Reply:
Whether COURSE_HOME_MICROFRONTEND is active or not doesn’t seem to matter anymore for this test, because rendering the outline tab would now always redirect to MFE (i.e. we will get a 404 response for legacy). So I removed the overriding COURSE_HOME_MICROFRONTEND part and the second argument to the function (whether mfe is active or not) as well.